demo/src/main/java/com/orange/ods/demo/ui/components/sheets/ComponentSheetsBottom.kt

```diff
@@ -53,7 +53,7 @@ fun ComponentSheetsBottom() {
             bottomSheetContent = {
                 when (content.value) {
                     SheetsBottomCustomizationState.Content.Empty -> {
-                        Box(modifier = Modifier.height(200.dp))
+                        Box(modifier = Modifier.height(168.dp))
```

Comment:
This works but the drawback is that we will have to change this value if the content of the bottom sheet changes (and also we are not 100% that the sheet will look the same on all screens).

I suggest this fix instead:

```kotlin
bottomSheetContent = {
    val isEmpty = content.value == SheetsBottomCustomizationState.Content.Empty
    val recipes = LocalRecipes.current
    recipes.take(3).forEach { recipe ->
        OdsListItem(
            Modifier
                .iconType(OdsListItemIconType.Icon)
                .alpha(if (isEmpty) 0.0f else 1.0f),
            icon = recipe.iconResId?.let { iconRes ->
                {
                    OdsListItemIcon(
                        painterResource(id = iconRes)
                    )
                }
            },
            text = recipe.title
        )
    }
}
```